README.md

```diff
@@ -3,122 +3,205 @@
 ```bash
-$ docker run -p 50000-50050:50000-50050/udp -p 4000:4000/tcp -e INTEGRATED_TURN_PORT_RANGE=50000-50050 -e INTEGRATED_TURN_IP=<IPv4 address> membrane_videoroom
+docker run <rest of the options...> -e INTEGRATED_TURN_CERT=/usr/local/key.cert -e INTEGRATED_TURN_PKEY=/usr/local/key.priv -v <path to the certificate on the host filesystem>:/usr/local/key.cert -v <path to the private key on the host filesystem>:/usr/local/key.priv membrane_videoroom
 ```
 
-Then go to <http://localhost:4000/>.
 
-## OpenTelemetry
-By default OpenTelemetry is turned off. You can turn it on by going to `config/runtime.exs` and changing `otel_state` to one of four possible values:
+### OpenTelemetry
+The videoroom provides observability metrics 
+By default, OpenTelemetry is turned off. You can turn it on by going to `config/runtime.exs` and changing `otel_state` to one of three possible values:
 * `:local` - OpenTelemetry traces will be printed on stdout
-* `:zipkin` - OpenTelemetry traces are sent to Zipkin. You can change the url traces are sent to in `config/runtime.exs`. To setup zipkin you can run this command `docker run -d -p 9411:9411 openzipkin/zipkin`.
+* `:zipkin` - OpenTelemetry traces are sent to Zipkin. You can change the url traces are sent to in `config/runtime.exs`. To set up zipkin you can run this command `docker run -d -p 9411:9411 openzipkin/zipkin`.
 * `:honeycomb` - OpenTelemetry traces are sent to Honeycomb. You have to specify "x-honeycomb-team", which is API KEY for this service.
 
+
+## FAQ
+Below you can find a list of Frequently Asked Questions (FAQ).
+In case of any errors occurring during the setup, we encourage you to get familiar with that list.
+If you haven't found an answer to your question, we invite you to ask it on [the Membrane's Discord server](https://discord.com/channels/464786597288738816/1007190795167731742).
+
+### Known bugs
+#### `(Mix) Could not compile dependency :fast_tls,`
+As the error suggests, there was a problem with the `:fast_tls` dependency compilation. It's usually caused by the fact, that the compiler cannot
+find OpenSSL dependencies in your system.
+In the setup instruction above we have set the compiler flags pointing to the appropriate directories containing OpenSSL dependencies on a given OS.
+However, if you have your OpenSSL installed in some custom location, you will need to change the compiler flags (i.e. [`LDFLAGS`, `CFLAGS`, `CPPFLAGS`] or [`C_INCLUDE_PATH`, `LIBRARY_PATH`], depending on your compiler and OS) and the `PKG_CONFIG_PATH` environment variable.
+For more instructions on how to install the `:fast_tls` dependency, please visit the [Fast TLS repository](https://github.com/processone/fast_tls).
+
+#### Why, after joining the room, I don't see a video stream from the others?
+Take a look at the console logs - most probably there are prints of the following form:
+```
+client error exporting spans {:failed_connect,
+[{:to_address, {'localhost', 4318}}, {:inet, [:inet], :econnrefused}]}
```

Comment:
This log is about OpenTelemetry, not about ICE. When ICE fails, potential error log on server is `ICE restart failed due to timeout`

Reply:
Hmm, in fact, the misconfiguration of the `EXTERNAL_IP` env causes only these error prints in my case - I don't see `ICE restart failed due to timeout` messages.

Reply:
Lack of setting value of `EXTERNAL_IP` env might cause ICE errors in some cases, but not always, especially when you try to connect Google Chrome with locally running server

Reply:
This error comes from one of the `opentelemetry` dependencies and is about inability to connect with service, that would be responsible for collecting opentelemetry spans